upgrade/network/chn_enable.md

```diff
@@ -0,0 +1,830 @@
+2. Update phase
+   1. Add the `CHN` data and configurations while the `CAN` remains as-is
+3. Migrate phase - perform a controlled configuration and migration of components from the `CAN` to the `CHN`
+      1. NCN workers
+      2. CSM services
+      3. UAN
+      4. UAI
+      5. Compute (optional)
+4. Cleanup phase
+   1. Remove the `CAN` from operations and all data sets
+5. Upgrade the management network switch configurations
+
+The procedure, to be safe and flexible, is intensive from both the number of steps involved and the amount of system data which needs to be managed.
+However, during the migration phase, ample time and flexibility exists to contact system users as well as reverse the migration.
+
+**Note** that updates to the CSM management network are at the very end of this procedure.  CSM 1.3 network updates consist only of critical bugfixes as well as interface and `ACL` changes. This completes Bifurcated `CAN` transitions begun in CSM 1.2.
+
+## Prerequisites
+
+1. The system must have successfully completed a CSM 1.3 upgrade or be ready for CSM 1.3 upgrade.
+2. [Install the latest CSM 1.3 documentation](../../update_product_stream/README.md#check-for-latest-documentation)
+3. A site-routable IPv4 subnet for the `CHN`.  Minimally this must be sized to accommodate:
+   1. Three IPs for switching, plus
+   2. The number of `NCN` workers on the system, plus
+   3. One IP for the API ingress gateway, plus
+   4. The number of `NCN` `UANs` on the system, plus
+   5. The maximum number of `UAIs` required, plus
+   6. IPs for any other services to be brought up dynamically on the `CHN`
+   7. **Note** A `/24` subnet is usually more than sufficient for small-to-medium sized systems with minimal `UAI` requirements.
+4. The Slingshot high speed network is configured and up, including the fabric manager service.  This network is required to transit CHN traffic.
+5. The Slingshot Host Software is installed and configured on NCN Worker nodes.  This is required to expose CHN services.  For the purpose of CHN, the host software creates the (required) primary IPv4 address on `hsn0`, often a `10.253` address.
+
+## Backup Phase
+
+### Preparation
+
+1. (`ncn-m001#`) Make working directories for the procedure.
+
+   ```bash
+   mkdir migrate_can_to_chn
+   cd migrate_can_to_chn
+   export BASEDIR=$(pwd)
+   mkdir backups updates cleanup
+   export BACKUPDIR=${BASEDIR}/backups
+   export UPDATEDIR=${BASEDIR}/updates
+   export CLEANUPDIR=${BASEDIR}/cleanup
+   ```
+
+2. (`ncn-m001#`) Obtain an API token.
+
+   ```bash
+   export TOKEN=$(curl -s -k -S -d grant_type=client_credentials -d client_id=admin-client \
+                 -d client_secret=`kubectl get secrets admin-client-auth -o jsonpath='{.data.client-secret}' | base64 -d` \
+                 https://api-gw-service-nmn.local/keycloak/realms/shasta/protocol/openid-connect/token | jq -r '.access_token')
+   ```
+
+### Create system backups
```

Comment:
Would be great if we could combine all of this into a single script.

Reply:
Given the limited audience and one-time nature of this I would prefer not to wrap this in a script.